mars/session.py

```diff
@@ -43,10 +43,9 @@ class Session(object):
 
     def __init__(self, endpoint=None):
         if endpoint is not None:
-            from .web import get_client
+            from .web.session import Session
```

Comment:
Here must be 3 kinds of sessions, besides the LocalSession, if the endpoint is a http/https url, the web session required, but for the LocalCluster, web is optional, if the endpoint is the scheduler address, the session should be based on the actor client api.

Reply:
A LocalClusterSession can be added to do the job.